scripts/create_project_table.py

```diff
@@ -0,0 +1,137 @@
+            metadata_lines = []
+            for line in lines[1:]:
+                if line.strip() == "---":
+                    break
+                metadata_lines.append(line)
+
+            metadata = yaml.safe_load("\n".join(metadata_lines))
+
+        return metadata
+
+    def get_title(self) -> str:
+        return self.metadata.get("title", None)
+
+    def get_description(self) -> str:
+        return self.metadata.get("description", "No description available.")
+
+    def get_integrations(self) -> str:
+        integrations = self.metadata.get("integrations", [])
+        return ", ".join(integrations) if integrations else "None"
+
+    def to_table_row(self) -> str:
+        title = self.get_title()
+        if not title:
+            return ""
+
+        description = self.get_description()
+        integrations = self.get_integrations()
+        relative_path = os.path.relpath(os.path.dirname(self.readme_file), FOLDER_PATH)
+        link = f"./{relative_path}/"
+
+        return f"| [{title}]({link}) | {description} | {integrations} |\n"
+
+
+def generate_readme_table(folder_path) -> List[str]:
+    """Generate a list of table rows from README metadata."""
+    readme_files = glob.glob(
+        os.path.join(folder_path, "**", "README.md"), recursive=True
+    )
+
+    rows = []
+    for readme_file in readme_files:
+        processor = ReadmeMetadataProcessor(readme_file)
+        row = processor.to_table_row()
+        if row:
+            rows.append(row)
+    return rows
+
+
+def insert_rows_to_table_start(readme_file, new_rows) -> None:
+    with open(readme_file, "r") as file:
+        lines = file.readlines()
+
+    try:
+        header_index = (
+            next(
+                i
+                for i, line in enumerate(lines)
+                if line.startswith("| Name") and "| Description" in line
+            )
+            + 2
+        )
+    except StopIteration:
+        print("No table header found.")
+        return
+
+    lines[header_index:header_index] = new_rows
+
+    with open(readme_file, "w") as file:
+        file.writelines(lines)
+
+
+def process_readme(file_path) -> None:
+    """
+    Clean the README file by removing old table rows while keeping the header.
+
+    This prepares the README for a new table to be added.
+    """
+    new_lines = []
+    keep_table_header = False
+
+    with open(file_path, "r") as file:
+        for line in file:
+            # Check if it's the table header or divider row
+            if line.strip().startswith("| Name") or line.strip().startswith("| :"):
+                keep_table_header = True
+                new_lines.append(line)
+                continue
+
+            if keep_table_header and line.strip().startswith("|"):
+                continue
+
+            new_lines.append(line)
+
+    with open(file_path, "w") as file:
+        file.writelines(new_lines)
+
+
+def main():
+    print("Starting README processing...")
```

Comment:
Once you know this code works, I'd remove all these prints - they're not really useful in general: the user doesn't need to know that the program started, or finished, or exactly how many files were processed. The user only cares if there are errors/warnings.